src/object/firefly.cpp

```diff
@@ -78,7 +79,10 @@ Firefly::draw(DrawingContext& context)
 
   if (m_sprite_name.find("torch", 0) != std::string::npos && (activated ||
         m_sprite->get_action() == "ringing")) {
-    m_sprite_light->draw(context.light(), m_col.m_bbox.get_middle() - TORCH_LIGHT_OFFSET, 0);
+    if (m_flip == NO_FLIP)
+      m_sprite_light->draw(context.light(), m_col.m_bbox.get_middle() - TORCH_LIGHT_OFFSET, 0);
+    else
+      m_sprite_light->draw(context.light(), m_col.m_bbox.get_middle() + TORCH_LIGHT_OFFSET, 0);
```

Comment:
What is the difference between those two lines?

Reply:
That 
![image](https://user-images.githubusercontent.com/14074789/147924696-27d406f4-44ac-43ee-8858-64adbd238aad.png)

Reply:
I'll replace it with a ternary conditional